R/query_dataset.R

```diff
@@ -0,0 +1,57 @@
+#' Query a data set
+#'
+#' @description
+#' A short description...
+#'
+#' @inheritParams api_url
+#' @param method An API query method. Needs to be "GET"
+#'
+#' @return Data frame containing query results
+#' @export
+#'
+#' @examples
+#' query_dataset(example_id(), indicators = example_id("indicator"), page = 1, page_size=10)
```

Comment:
We should have the first examples for this as ones without the page arguments as that's how I'd expect most to use it

Reply:
Done